releasenotes/index.rst

```diff
@@ -13,16 +13,17 @@ see the Release Notes for it.
 Current Stable
 ==============
 
-* :ref:`Dojo Toolkit v1.11 Release Notes <releasenotes/1.11>`
+* :ref:`Dojo Toolkit v1.12 Release Notes <releasenotes/1.12>`
 
 Current Development
 ===================
 
-* :ref:`Dojo Toolkit v1.12 Release Notes <releasenotes/1.12>`
+* :ref:`Dojo Toolkit v1.13 Release Notes <releasenotes/1.13>`
```

Comment:
Added `1.13.rst` with ToC and sections as taken from `1.12.rst` and prior notes.